src/Styles/Hooks.php

```diff
@@ -9,7 +9,8 @@
 
 class Hooks implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC_Action {
 
-	const META_KEY = '_cs_generated_styles';
+	const META_KEY_OLD = '_cs_generated_styles';
+	const META_KEY_V6 = '_cs_generated_tss';
```

Comment:
This is the new post meta key for the styles (I am not 100% sure it was changed in v6 though, but I had to find a name...).